src/components/interfaces/MOBILE_API.xml

```diff
@@ -3232,12 +3555,6 @@
         </param>
     </struct>
 
-    <struct name="RGBColor">
```

Comment:
Why move this struct?

Reply:
it is moved to a different place in file.

Reply:
I am aware, I was asking why it was moved

Reply:
@jacobkeeler That was moved because this struct used in a struct `LightState` which is above and during building it was crashed with 'unknown name RGBColor` error.